aries-site/src/data/color.js

```diff
@@ -10,40 +10,40 @@ const { colors } = theme.global;
 /* Color Palette */
 const primaryColors = [
   {
-    name: 'green',
+    name: 'Green',
```

Comment:
@britt6612 @gregfuruiye We are inconsistent with our casing here. In some places we lowercase, in other we are capitalizing. I think we need to be consistent throughout.

Throughout the majority of the page we are lowercased (e.g. text, text-strong, etc.). Additionally, code references will be lowercased.

Reply:
@gregfuruiye I changed them back to lower case I think Matt makes a good comment with having the code references lowercase maybe we should keep it the same so that it doesn't cause more confusion.